site/public/js/admin-gradeable-updates.js

```diff
@@ -107,6 +138,8 @@ $(document).ready(function () {
 
     ajaxCheckBuildStatus();
     $('input:not(#random-peer-graders-list,#number_to_peer_grade),select,textarea').change(function () {
+        updateDueDate();
+        updateReleaseDate();
```

Comment:
Is there any risk to having this run on changing the inputs for non-date fields? Do we even want that, and could we make this only operate on the date fields / `has_due_date_no`?

Reply:
I'm not sure if there is any risk but I can wrap it in an if statement to make sure we are targeting those specific radio buttons since that is when it matters. I just wanted to make sure these were called before the data was being sent.